packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -0,0 +1,88 @@
+import { LitElement, customElement, html, property } from 'lit-element';
+// import { Base } from '@sfx/base';
+import '@sfx/autocomplete';
+import { SAYT_EVENT } from './events';
+
+/**
+ * The `sfx-sayt` component is responsible for displaying and hiding the
+ * `sfx-autocomplete` and `sfx-products` components.
+ */
+@customElement('sfx-sayt')
+// TODO: look into the Base class so this class can extend it again. The
```

Comment:
This comment belongs in a ticket, not here. It's also vague and out of context enough that I don't know what the problem is just by reading it, so I don't get anything useful out of this comment.